l10n_fi_invoice/views/account_move_templates.xml

```diff
@@ -0,0 +1,535 @@
+
+                        .payment-terms {
+                        font-size: .7em;
+                        margin-top: .5em;
+                        line-height: 1em;
+                        }
+
+                        .payment-terms p {
+                        margin-bottom: 0;
+                        }
+
+                        .invoice_tbody tr td:last-child, .products-table-totals tr td:last-child, .table-condensed tr td:last-child {
+                        width: 160px;
+                        }
+                    </style>
+                    <div style="height: 2.62cm; position: relative;">
+                        <div class="padding-correction" style="position: absolute; width: 100%; bottom: 0;">
+                            <div>
+                                <div class="col-2 offset-1" style="float: none; display: inline-block; vertical-align: middle;">
+                                    <img t-if="o.company_id.logo" t-att-src="image_data_uri(o.company_id.logo)" style="max-height: 2.1cm; max-width: 4cm;"/>
+                                </div>
+                                <div class="col-4" style="font-size: .85em; float: none; display: inline-block; vertical-align: middle;">
+                                    <div t-field="o.company_id.partner_id" t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;], &quot;no_marker&quot;: true}"/>
+                                </div>
+                                <div class="col-5 text-left" style="font-weight: bold; font-size: large; float: none; display: inline-block; vertical-align: bottom; width: auto; padding-left: 10px;">
+                                    <span t-if="o.type == 'in_refund'">Refund</span>
+                                    <span t-if="o.type == 'out_refund'">Refund</span>
+                                    <span t-if="o.type == 'in_invoice'">Invoice</span>
+                                    <span t-if="o.type == 'out_invoice'">Invoice</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="padding-correction" style="min-height: 6.68cm">
+                        <!-- Min height reduced to make invoice fit on one page. Original height was 8.68cm" -->
+                        <div class="row">
+                            <div class="col-6 offset-1" style="margin-top: 3em; margin-bottom: 4em; font-size:1.1em;">
+                                <address t-field="o.partner_id" t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;], &quot;no_marker&quot;: true}"/>
+                                <span t-if="o.partner_id.vat">VAT:
+                                    <span t-field="o.partner_id.vat"/>
+                                </span>
+                                <span id="partner_business_id"/>
```

Comment:
From which module does this field come from?

Reply:
It's coming from `base`

Reply:
`grep` disagrees. It doesn't seem to be an Odoo field.

Reply:
Oh, you mean the lowest line. It's just span with an id. It's not an Odoo field. 

Seems like `l10n_fi_invoice_business_code` used it to add additional info to the prints.